packages/web/src/dev/DevelopersPage.tsx

```diff
@@ -0,0 +1,47 @@
+import * as React from 'react'
+import { StyleSheet, View } from 'react-native'
+import DeveloperPhoneCover from 'src/dev/DeveloperPhoneCover'
+import { Contribute, EngageAsDeveloper } from 'src/dev/Engage'
+import Features from 'src/dev/Features'
+import FullStack from 'src/dev/FullStack'
+import Sandbox from 'src/dev/Sandbox'
+import { H2, H3 } from 'src/fonts/Fonts'
+import OpenGraph from 'src/header/OpenGraph'
+import { I18nProps, NameSpaces, withNamespaces } from 'src/i18n'
+import ConnectionFooter from 'src/shared/ConnectionFooter'
+import menuItems from 'src/shared/menu-items'
+import { standardStyles } from 'src/styles'
+const previewImage = require('src/dev/dev-og.png')
```

Comment:
Why require here? Also, spacing

Reply:
at one point i found that import and images don't always work out and it got me stuck in this habit  even if its not true anymore